js/capture.js

```diff
@@ -135,7 +135,7 @@ const capture = {
         target: targetUrl.hostname,
         origin: originUrl.hostname,
         requestTime: response.timeStamp,
-        firstParty: false
+        firstParty: 0
```

Comment:
Okay I kept all boolean data types as is and only change them in the `setWebsite` method in store with a single comment about it.